### ISSUE TITLE
[RFC] Contextual release news items on bolt.cm

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#6489]
## Background 
A common bit of feedback, especially with people new/unfamiliar with the project, is _"Your current version says 3.2.9, but the news item on the bolt.cm front page, and my admin news feed says 'Latest news: Bolt 3.2.5 released.' What gives?"_

A quick look at our evolving release news items, shows:
![release note comparison](https://cloud.githubusercontent.com/assets/1427081/24032741/cf4364ec-0ae9-11e7-96b7-97446ef01108.png)

The section _"Installation and Upgrade"_ is static, and the introduction for all in-tents & porpoises can be too. 

## Proposal 
1. Have one news item per minor release
2. Update said news item with each new patch release

e.g.
![release note rfc](https://cloud.githubusercontent.com/assets/1427081/24033275/68064602-0aec-11e7-81e7-401d30b3ce40.png)

**NOTE:** Above is an indication of the danger of Gawain + HTML + CSS + GIMP … obviously just PoC.

In my mind at least — we all know how well that's doing right now :stuck_out_tongue_winking_eye: — it will always show the current "minor", which is correct, and also save on the time to produce new items for patch versions.

Basically the whole thing becomes and editor/designer/novice friendly change log (notable changes are drawn from `CHANGELOG.md` anyway)

I thought this was easier to put as a short lived RFC, closing on 24 March sounds GTM.

**Comment by bobdenotter** [comment]
There was some work done by Nestor and me, to make something like this easier to manage. I'll dig up what the status was, patch it up and incorporate it into the workflow better..

**Comment by GawainLynch** [comment]
> There was some work done by Nestor and me, to make something like this easier to manage. I'll dig up what the status was, patch it up and incorporate it into the workflow better..

@bobdenotter I know we have bigger fish to fry right now, but just in case this is _easy_ … "reminder" :smile:

**Comment by GawainLynch** [comment]
@bobdenotter this is relevant to what we were discussing this morning in #development … If you've got this in hand for the upcoming release, can you send this RFC to the issue place in the sky please :+1:

**Comment by GawainLynch** [comment]
Closing as I believe that we're agreed on this moving forward :champagne:
